test/studies/shootout/submitted/fasta2.chpl

```diff
@@ -92,6 +92,8 @@ proc repeatMake(desc, alu, n) {
   const r = alu.size,
         s = [i in 0..(r+lineLength)] alu[i % r];
 
+  //  writeln(s.domain);
```

Comment:
Are some of the changes in this file intentional?

Reply:
Shoot, no.  I hadn't looked at the diff yet myself and forgot that I had outstanding changes hanging out in my directory...